src/compiler/transformers/add-component-meta-static.ts

```diff
@@ -2,7 +2,20 @@ import type * as d from '../../declarations';
 import { convertValueToLiteral, createStaticGetter } from './transform-utils';
 import ts from 'typescript';
 
-export const addComponentMetaStatic = (cmpNode: ts.ClassDeclaration, cmpMeta: d.ComponentCompilerMeta) => {
+/**
+ * Update an instance of TypeScript's IR for a class declaration
```

Comment:
Nit - can we update the first usage of this acronym to include its definition for future readers?
```suggestion
 * Update an instance of TypeScript's Intermediate Representation (IR) for a class declaration
```

Reply:
yeah I think that's a good idea to do across the board 👍